src/main/java/Duke.java

```diff
@@ -1,10 +1,147 @@
+import taskitems.Greet;
+import taskitems.exceptions.IllegalInputException;
+import taskitems.TaskManager;
+
+
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
 public class Duke {
+
+    static Greet greet;
+    static Scanner reader = new Scanner(System.in);
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        TaskManager taskManager = new TaskManager();
+        try {
+            taskManager.loadData();
+        } catch (FileNotFoundException f) {
+            System.out.println("No saved data found");
+        }
+        greet.sayHi();
+        taskLoop(taskManager);
+        //echo();
+        greet.sayBye();
+    }
+
+    public static void taskLoop(TaskManager taskManager){
```

Comment:
This method is too long ([>30 LoC](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods)). How about abstracting out code for each of cases below and applying [SLAP](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#slap-hard) here?